build/configs/default_config.yaml

```diff
@@ -119,6 +119,10 @@ p2p:
   port: 38202
   # Timeout when opening connections to peers
   connectTimeout: "30s"
+  # minimum delay for every p2p response to avoid timing attacks
+  # adjust based on host resources (SSD, CPU, cores ...)
+  # Look in logs for: "Time consumed by operation" if x=(valueRead * 2) is less than value below, then change responseDelay to x
+  responseDelay: "500ms"
```

Comment:
Did you measure 500ms? I think that's pretty short as a default. We should also explain why the delay is needed and the privacy implications of setting it too low.

Reply:
I did perform a benchmark with an SSD and with HDD, resulting in the above for HDD. For SSD:
```
- Document Exist on a ~1.2M DB:
  - MessageTypeRequestSignature: ~11ms
  - MessageTypeSendAnchoredDoc: ~13ms
  - MessageTypeGetDoc: ~5ms

- Document Does not exist on a ~1.2M DB:
  - MessageTypeRequestSignature: ~11ms
  - MessageTypeSendAnchoredDoc: ~13ms
  - MessageTypeGetDoc: ~1.5ms (*timing attack)
```
This by having 1.2 Million records in DB. You can see the significant jump between both host configurations, it is hard to set a static delay that work for all. That is why is configurable. I explain the delay in the comment, we can add more details to it in the docs.